entrypoint.sh

```diff
@@ -151,7 +151,7 @@ fi
 
 if [[ "${INI}" != "" ]];then
     echo "Installing php.ini settings"
-    echo $INI > /etc/php/${PHP}/cli/conf.d/99-settings.ini
+    echo $INI | sed "s/{NL}/\n/g" > /etc/php/${PHP}/cli/conf.d/99-settings.ini
```

Comment:
this is a super dumb "fix", now values can't contain `{NL}`!
bash has no problems containing newlines!

sending PR with actual fix with quoting echo arguments!

Reply:
https://github.com/laminas/laminas-continuous-integration-action/pull/16/files

Reply:
1st of all: there is nothing dumb when not having *all* scenarios in mind
2nd: you might want to learn how to communicate such problems